neuralprophet/benchmark.py

```diff
@@ -232,7 +234,25 @@ def __post_init__(self):
             "SMAPE": _calc_smape,
         }
 
-    def _evaluate_model(self, model, df_train, df_test):
+    def store_results_to_csv(self, fcst_train, fcst_test, save_dir, current_fold):
+        # save fcst and create dir if necessary
+        if save_dir:
+            folder_exist = os.path.isdir(save_dir)
+            if not folder_exist:
+                os.makedirs(save_dir)
+            filename_ext = (
```

Comment:
why not use experiment name?

Reply:
experiment_name wraps the necessary data into a dict, where the key `params` is a dict converted to a  string. 
It's cleaner in my opinion, to directly grab the dict `params` and access the keys from there. Hmu if you don't know what I mean.

Reply:
I agree. but we might want to also use this elsewhere. 
Let's add this string creation to __post_init__ 
Let's do this:
rename `experiment_name` to `experiment_metadata`
store this string you are creating as `experiment_name`